scripts/archive_leg_to_csv.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+import csv
+import django
+from django import conf
+import click
+from utils import (
+    get_jurisdiction_id,
+)
+
+
+def init_django():  # pragma: no cover
+    conf.settings.configure(
+        conf.global_settings,
+        SECRET_KEY="not-important",
+        DEBUG=False,
+        INSTALLED_APPS=(
+            "django.contrib.contenttypes",
+            "opencivicdata.core.apps.BaseConfig",
+            "opencivicdata.legislative.apps.BaseConfig",
+        ),
+        DATABASES={
+            "default": {
+                "ENGINE": "django.contrib.gis.db.backends.postgis",
+                "NAME": "openstatesorg",
+                "USER": "openstates",
+                "PASSWORD": "openstates",
+                "HOST": "localhost",
+                "PORT": "5405"
+            }
+        },
+        MIDDLEWARE_CLASSES=(),
+    )
+    django.setup()
+
+
+@click.command()
+@click.argument("state_abbr", nargs=-1)
+@click.argument("session", nargs=1)
```

Comment:
looking good, it'd be great to have the ability to run this for all sessions and spit out a file per session

Reply:
Script will now generate CSVs for all of a state's sessions if none provided.